source/sass/glyphs.scss

```diff
@@ -1,26 +1,43 @@
-@mixin glyph($id, $width: 20px, $height: 20px) {
+@mixin glyph($id, $buyers-guide: false, $width: 20px, $height: 20px) {
   content: "";
   display: inline-block;
   width: $width;
   height: $height;
   margin-right: 20px;
-  background: url(../_images/glyphs/#{$id}.svg) no-repeat 0 0 / contain;
+
+  @if $buyers-guide {
+    background: url(../_images/buyers-guide/#{$id}.svg) no-repeat 0 0 / contain;
+  } @else {
+    background: url(../_images/glyphs/#{$id}.svg) no-repeat 0 0 / contain;
+  }
 
   @media screen and (min-width: $bp-lg) {
     margin-right: 10px;
   }
 }
 
-@mixin hoverGlyph($id) {
+@mixin hoverGlyph($id, $buyers-guide: false) {
   &:hover::before,
   &:focus::before {
-    background: url(../_images/glyphs/#{$id}.svg) no-repeat 0 0 / contain;
+    @if $buyers-guide {
+      background: url(../_images/buyers-guide/#{$id}.svg)
+        no-repeat
+        0
+        0 /
+        contain;
+    } @else {
+      background: url(../_images/glyphs/#{$id}.svg) no-repeat 0 0 / contain;
+    }
   }
 }
 
-@mixin compositeGlyph($id) {
+@mixin compositeGlyph($id, $buyers-guide: false) {
```

Comment:
Instead of having the 2nd parameter being a `boolean`, let's make it a string called `path-to-directory` with default value `../_images/glyphs/`. This way more projects/apps on Foundation site will be able to utilize these mixins better.